src/core_plugins/kibana/public/management/sections/settings/components/search/search.test.js

```diff
@@ -55,4 +55,27 @@ describe('Search', () => {
     component.find('input').simulate('keyup', { target: { value: 'new filter' } });
     expect(onQueryChange).toHaveBeenCalledTimes(1);
   });
+
+  it('should handle query parse error', async () => {
+    const onQueryChangeMock = jest.fn();
+    const component = mountWithIntl(
+      <Search.WrappedComponent
+        query={query}
+        categories={categories}
+        onQueryChange={onQueryChangeMock}
+      />
+    );
+
+    // Send invalid query
+    component.find('input').simulate('keyup', { target: { value: '?' } });
```

Comment:
Is it possible to use a `data-test-subj` selector the way you did in the other test?